hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/holder/PagingPredicateHolder.java

```diff
@@ -39,17 +43,24 @@ public class PagingPredicateHolder {
     private final int pageSize;
     private final int page;
     private final byte iterationTypeId;
-    private final Data partitionKeyData;
+    private final Collection<Data> partitionKeysData;
 
-    public PagingPredicateHolder(AnchorDataListHolder anchorDataListHolder, Data predicateData, Data comparatorData, int pageSize,
-                                 int page, byte iterationTypeId, Data partitionKeyData) {
+    public PagingPredicateHolder(AnchorDataListHolder anchorDataListHolder, Data predicateData, Data comparatorData,
+                                 int pageSize, int page, byte iterationTypeId,
+                                 Data partitionKeyData,
+                                 boolean partitionKeysDataExists,
```

Comment:
`partitionKeysDataExists` is not used?

Reply:
No it's not, but it has to be there as this is called by auto generated code